### DISCUSSION TITLE
Projects - Description

### DISCUSSION THREAD

**Discussion by Mikeroboct** [community/community#8040] [Projects and Issues] [answered]
It would be nice when you define a "Project" that a write up readme (or md) can go along with that Project to describe what the scope is. As an example , I may have different "projects" that has a set of capabilities that I would like to call out.

**Comment by mansona** [comment]
I would expand on this a bit more too. We are using the old-style projects currently for "initiatives" in our open source project and we have the description field that defines a) the scope of the project and b) the champions of the project. 

Having only a name field available limits the sorts of metadata that we can assign to a project.

**Comment by azenMatt** [comment, marked as answer]
We just shipped some updates here - https://github.blog/changelog/2022-02-02-the-new-github-issues-02-02-update/ - does that help solve this for you?
